package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lucid-ui",
-  "version": "2.15.1",
+  "version": "2.15.2-0",
```

Comment:
Yeah it's a prerelease since I needed to test it in `anx-react`. It'll get blown incremented to a proper version on the next release.